src/Pages/garamsong/Main/components/FeedsBox.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Feed from './Feed';
+import './FeedsBox.scss';
+
+class FeedsBox extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      commentList: [],
+      newComment: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/garamsong/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(comments => {
+        this.setState({
+          commentList: comments,
+        });
+      });
+  }
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/garamsong/commentData.json')
      .then(res => res.json())
      .then(commentList => this.setState({ commentList }));
  }
```
- 객체 단축 속성명을 이용하면 객체를 조금 더 깔끔하게 표현할 수 있습니다
- public 폴더에서 가져오는 데이터의 경우 url과 포트를 생략할 수 있습니다.
- get method는 fetch api 기본 값이므로 역시 생략해줄 수 있습니다

Reply:
오오,, 객체 단축 속성명을 이용한 부분이 와닿지 않았었는데 실제 코드에 적용해보고 이해가 되는것 같습니다! 종택님이 말씀하시는건 두번째.then 뒤에 인자로 받아오는 부분을 실제 state의 값의 key값인 commentList와 동일하게 설정하면 굳이 길게 쓰지 않아도 해당 인자를 내가 원하는 key값에 넣을 수 있다는 말씀 맞겠죠?